frontend-nannyway/src/components/footer/FooterButton.js

```diff
@@ -0,0 +1,17 @@
+import Button from "@material-ui/core/Button";
+import React from "react";
+
+export default function FooterButton({colorStyle, onClickAction, variant, classNameButton, startIcon, size, text}) {
+    return (
+        <Button
+            onClick={onClickAction}
+            color={colorStyle}
+            variant={variant}
+            className={classNameButton}
+            startIcon={startIcon}
+            size={size}
+        >
+            {text}
+        </Button>
```

Comment:
So far you could achieve the same effect by just using MUI's `<Button>` in your `<Footer>`.
Are you planning to further customize this component? If so, I recommend mentioning it in the PR request description.